packages/contracts/contracts/interpreters/TwoPartyFixedOutcomeETHInterpreter.sol

```diff
@@ -5,7 +5,7 @@ import "../interfaces/Interpreter.sol";
 import "../libs/LibOutcome.sol";
 
 
-contract TwoPartyEthAsLump is Interpreter {
+contract TwoPartyFixedOutcomeETHInterpreter is Interpreter {
```

Comment:
why the name change? seems slightly worse IMO

Reply:
- `Lump` is never used in any other context
- `ETH` should always be capitalized
- The outcome type is `TwoPartyOutcome` not `TwoParty`
- Standardizes the interpreter contract name as `<OutcomeType><AssetType>Interpreter`

I think it is much better